aten/src/ATen/native/cuda/CUDAJitLoops.cuh

```diff
@@ -156,6 +156,8 @@ void launch_jitted_vectorized_kernel(
     at::cuda::jit::launch_jitted_pwise_function(
         *fn_ptr, args.data(), {grid, 1u, 1u}, {num_threads(), 1u, 1u});
   } else {
+#pragma nv_diagnostic push
+#pragma nv_diag_suppress 177
```

Comment:
Add a comment about what warning we are suppressing here and why?

Reply:
Added in commit 0cadec0